cluster/images/etcd-empty-dir-cleanup/Dockerfile

```diff
@@ -12,12 +12,15 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM gliderlabs/alpine
+FROM busybox
+
+COPY etcdctl etcd-empty-dir-cleanup.sh /
+RUN chmod a+rx /etcdctl /etcd-empty-dir-cleanup.sh
 
-RUN apk-install bash
-ADD etcd-empty-dir-cleanup.sh etcd-empty-dir-cleanup.sh
-ADD etcdctl etcdctl
 ENV ETCDCTL /etcdctl
 ENV SLEEP_SECOND 3600
-RUN chmod +x etcd-empty-dir-cleanup.sh
-CMD bash /etcd-empty-dir-cleanup.sh
+
+# nobody:nobody
+USER 65534:65534
```

Comment:
`etcdctl` doesn't seem to care which user it runs as.

I tested this, and it seems to work, except that I think [this conditional](https://github.com/kubernetes/kubernetes/pull/41674/files#diff-020acfcee2e33a66240f5189cafbcecaR20) is buggy, possibly under `ash`, possibly in general. digging some more...